app/fastlane/Fastfile

```
@@ -66,7 +71,7 @@
         file_path: result[:ipa_path],
         channel_id: ENV["SLACK_CHANNEL_ID"],
         initial_comment: "🍎 iOS v#{package_version} (Build #{result[:build_number]}) deployed to TestFlight",
-        title: "#{PROJECT_NAME}-#{package_version}-#{result[:build_number]}.ipa",
+        title: "#{APP_NAME}-#{package_version}-#{result[:build_number]}.ipa",
```

RuboCop (1.75.5)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)